app/design/frontend/Trubetskaia/Veta/Magento_LayeredNavigation/templates/layer/custom-layer-block.phtml

```diff
@@ -0,0 +1,4 @@
+<div id="custom-promo">
```

Comment:
Please use underscores (custom_layer_block.phtml) instead of dashes in template names.
This is the common naming convention in Magento, please try searching for template names in Magento core:
https://github.com/magento/magento2/tree/2.2/app/code/Magento/CheckoutAgreements/view/frontend/templates
see the examples. Just rename this template to custom_layer_block.phtml and fix it's path in other files.

Reply:
fixed. thank you